tenets/codelingo/go/writes-to-async-pointer/example.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+)
+
+type thing struct {
+	name string
+}
+
+func getNameFromPointer(t *thing) {
```

Comment:
`// readVulnerableField accesses data which may be vulnerable to concurrent writes`?